terraform/variables.tf

```diff
@@ -0,0 +1,92 @@
+variable "tariff_backend_secret_key_base" {
```

Comment:
Are we going to store these with the service? Maybe worth making them SSM parameters and passing them as service secrets when they're things like this - otherwise they get stored in plaintext